addusers.py

```diff
@@ -336,7 +337,7 @@ def parse_rows(rows):
     session = session.Session(auth=auth)
     
     openstack = Openstack(session=session, nova_version=nova_version, setpass_url=setpass_url)
-    auth_file = config.get("excelsheet", "auth_file")
+    auth_file = get_absolute_path(config.get("excelsheet", "auth_file"))
     worksheet_key = config.get("excelsheet", "worksheet_key")
```

Comment:
Oh, yes, sorry that I thought I put this below that line. I see. You need to have `get_absolute_path` for the files. Thanks for explaining it.